test/remote/gateways/remote_merchant_e_solutions_test.rb

```diff
@@ -22,7 +22,10 @@ def setup
         zip: '55555',
         phone: '555-555-5555',
         recurring_pmt_num: 11,
-        recurring_pmt_count: 10
+        recurring_pmt_count: 10,
+        card_on_file: 'Y',
+        cit_mit_indicator: 'C101',
+        account_data_source: 'Y'
```

Comment:
Can we add some tests to check the behavior of adding these fields to the request?